src/client/views/UpdatePattern.tsx

```diff
@@ -61,18 +61,22 @@ const UpdatePattern = (props: UpdatePatternProps) => {
 			content: string;
 			author_id: string;
 			id: string;
-			tags: Tags;
 		} = {
 			content: pattern.content,
 			author_id: pattern.author_id,
 			id,
-			tags: selectedTags,
 		};
 		console.log(`DTO`, patternDTO);
-		// noteService
-		// 	.updatePattern(id, patternDTO)
-		// 	.then(() => navigate(`/patterns/${id}`));
-		// .catch(e => Toast.error(e.message));
+		noteService.updatePattern(id, patternDTO);
+
+		const pattern_id: number = parseInt(pattern.id);
+		const tag_ids: number[] = [];
+		for (let tag of selectedTags) {
+			tag_ids.push(tag.id);
+		}
+		patternTags
+			.addNewTag({ pattern_id, tag_ids })
```

Comment:
```suggestion
			.addNewTags({ pattern_id, tag_ids })
```
Now that this has been changed, I'd recommend renaming this function!

Reply:
Renamed, thank you for catching that, I didn't realize that had the capability of adding multiple